tools/quantize/quantize-inc.py

```diff
@@ -78,6 +78,9 @@ def __len__(self):
     mp_palmdet=Quantize(model_path='../../models/palm_detection_mediapipe/palm_detection_mediapipe_2022may.onnx',
                         config_path='./inc_configs/mp_palmdet.yaml',
                         custom_dataset=Dataset(root='../../benchmark/data/palm_detection', dim='hwc', swapRB=True, mean=127.5, std=127.5, toFP32=True)),
+    mp_handpose=Quantize(model_path='../../models/handpose_estimation_mediapipe/handpose_estimation_mediapipe_2022may.onnx',
+                         config_path='./inc_configs/mp_handpose.yaml',
+                         custom_dataset=Dataset(root='../../benchmark/data/palm_detection', dim='hwc', swapRB=True, mean=127.5, std=127.5, toFP32=True)),
```

Comment:
I think we do not need this parameters `swapRB=True, mean=127.5, std=127.5`, since there are no such processings in pre-processing.

Reply:
The dataset used here is the same as the palm detection model, and if the parameters are removed, the quantified model is completely unusable.